gui/src/app/core/Scripting/Takeout/makePyRuntime.ts

```diff
@@ -0,0 +1,60 @@
+import { ProjectDataModel } from "@SpCore/Project/ProjectDataModel";
+import indent from "@SpUtil/indent";
+
+import spPreamble from "./py/preamble.py?raw";
+import spRunData from "./py/run_data.py?raw";
+import spLoadConfig from "./py/load_args.py?raw";
+import spCmdStan from "./py/cmdstan.py?raw";
+import spRunSampling from "./py/sample.py?raw";
+import spDrawsScript from "../pyodide/sp_load_draws.py?raw";
+import spRunAnalysis from "./py/run_analysis.py?raw";
+
+const makePyRuntimeScript = (project: ProjectDataModel) => {
+  const hasDataJson = project.dataFileContent.length > 0;
+  const hasDataPy = project.dataPyFileContent.length > 0;
+  const hasAnalysisPy = project.analysisPyFileContent.length > 0;
+
+  let script = `TITLE=${JSON.stringify(project.meta.title)}\n` + spPreamble;
+
+  // arguments
+  script += `argparser.add_argument("--install-cmdstan", action="store_true", help="Install cmdstan if it is missing")\n`;
+  if (hasDataJson && hasDataPy) {
+    script += `argparser.add_argument("--ignore-saved-data", action="store_true", help="Ignore saved data.json files")\n`;
+  }
```

Comment:
I think for simplicity's sake it's fine to have this parameter exist even if it can't do anything effective?